ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/KsqlResource.java

```diff
@@ -301,7 +297,7 @@ private SourceDescription describe(String statementText, String name) throws Exc
 
     StructuredDataSource dataSource = ksqlEngine.getMetaStore().getSource(name);
     if (dataSource == null) {
-      throw new Exception(String.format("Could not find data stream/table '%s' in the metastore",
+      throw new Exception(String.format("Could not find STREAM/TABLE '%s' in the Metastore",
```

Comment:
I know this was here already, but throwing `Exception` - really? above too

Reply:
i think it propagates to the client where it gets logged